parity/cli/mod.rs

```diff
@@ -199,13 +199,13 @@ usage! {
 			or |c: &Config| otry!(c.mining).notify_work.clone().map(|vec| Some(vec.join(","))),
 
 		flag_stratum: bool = false,
-			or |c: &Config| otry!(otry!(c.mining).stratum).enabled.clone(),
+			or |c: &Config| Some(c.stratum.is_some()),
```

Comment:
this will never fall back on the default if I understand it correctly; maybe there shouldn't be one?